Dockerfile

```diff
@@ -1,15 +1,21 @@
 
 ###################################
 #Build stage
-FROM golang:alpine AS build-env
+FROM alpine:3.7 AS build-env
 
 ARG GITEA_VERSION
 ARG TAGS="sqlite"
 ENV TAGS "bindata $TAGS"
 
+#Temporary in wait of golang:X.XX-alpine-3.7
```

Comment:
We `wait for` something, not `wait of`. ;) Not a blocker.

Reply:
Should be release monday on docker hub. Could be merge before and fix after. ^^